### ISSUE TITLE
Tab Content

### ISSUE THREAD

**Issue by imcodingideas** [AlgusDark/bloomer#102]
Hello,

Which component do I need to use to set a is active flag for tab content? Say I have 2 tabs, and those tabs have different content. 

I don't see any examples in the documentation.

**Comment by jonasgrunert** [comment]
You can give the actual Tab Component an active flag. The content should get marked active by itself as long as it is text. Hope that helps otherwise is [this link](https://bloomer.js.org/#/documentation/components/tabs) maybe helpful.

**Comment by imcodingideas** [comment]
@jonasgrunert yes thats to show the active tab but not displaying the content thats in that tab.

**Comment by jonasgrunert** [comment]
Sorry for the super late answer. To clarify: Tab just provides you with a menu. To render contents based on the active Tab is your responsibility. If you need additional information I can provide you with an example later on.

**Comment by AlgusDark** [comment]
@imcodingideas sorry for the late answer. I don't have internet at home and I cannot work on the project for some time (I was working on V2 and I'm going to continue again maybe in two months).

You can see the source code for the [Documentation page](https://github.com/AlgusDark/bloomer/blob/master/docs/src/Scenes/Documentation/Documentation.tsx) on how we do the change with React Router.

Here is [an example](https://codesandbox.io/s/93r05jm4jp) on how (_very quick and dirty_), you can make something similar without the need of React-Router. Remember that Bloomer has the same ideology like Bulma, while the don't provide JS functionality, we don't provide the functionality at all.
